.github/workflows/integration.yml

```diff
@@ -168,9 +168,9 @@ jobs:
           BIGQUERY_TEST_SERVICE_ACCOUNT_JSON: ${{ secrets.BIGQUERY_TEST_SERVICE_ACCOUNT_JSON }}
           BIGQUERY_TEST_ALT_DATABASE: ${{ secrets.BIGQUERY_TEST_ALT_DATABASE }}
           BIGQUERY_TEST_NO_ACCESS_DATABASE: ${{ secrets.BIGQUERY_TEST_NO_ACCESS_DATABASE }}
-          DBT_TEST_USER_1: user:buildbot@dbtlabs.com
-          DBT_TEST_USER_2: user:buildbot@fishtownanalytics.com
-          DBT_TEST_USER_3: group:dev-core@dbtlabs.com
+          DBT_TEST_USER_1: group:buildbot@dbtlabs.com
+          DBT_TEST_USER_2: group:dev-core@dbtlabs.com
+          DBT_TEST_USER_3: user:someone@dbtlabs.com # TODO
```

Comment:
TODO: Need to make these updates in a separate PR against `main`, so that they can be picked up in this branch (a la #216)

Reply:
https://github.com/dbt-labs/dbt-bigquery/pull/217 is intended to resolve this TODO.

Reply:
Going to change the base to `dbeatty/grantee-env-vars-take-2` and try rerunning tests!